01Strings/String/Tester+Strings.swift

```diff
@@ -44,11 +54,23 @@ extension Tester {
      one -> oneyay
      */
```

Comment:
what about uppercase vowels
easier to use a string "aeiouAEIOU".contains( firstLetter )